README.md

```diff
@@ -20,10 +20,15 @@ Install virtualenv:
     
 ## Generate the Lambda package
 
+Make sure your current branch is clean and up-to-date, and run the shell script `package-to-zip.sh`. You can pass an optional output filename to it as an argument.
+This script will create a .zip file in the directory you run it in.
+
+Alternately, here are the steps to create the .zip file manually:
+
 1. CD to the folder that contains main.py.
 1. Run the following command:
-`pip install -r requirements.txt -t `pwd``
-1. Compress the contents of folder (not the folder).
+`pip install -r requirements.txt -t .`
```

Comment:
If we plan on pushing these changes up stream, I'm not sure we want to change these instructions

Reply:
I wasn't going to push this PR upstream. It won't really be necessary if we're not maintaining this repo.